std/php/Boot.hx

```diff
@@ -96,7 +96,8 @@ class _hx_array implements ArrayAccess, IteratorAggregate {
 	}
 
 	function remove($x) {
-		for($i = 0; $i < count($this->a); $i++)
+		$len = count($this->a);
```

Comment:
I think this `count` is not necessary  because `$this->length` should always be equal to `count($this->a)` in this context.